R/gs_design_combo.R

```diff
@@ -341,3 +339,27 @@ gs_design_combo <- function(
 
   return(output)
 }
+
+#' Function to calculate power
```

Comment:
Could we include some additional description for this help function? While I understand that it is an internal function and an immediate documentation PR may not be necessary, it would still be beneficial to have it.

"This function calculates the difference between the derived power and the targeted power (1 - beta), based on the provided sample size, upper and lower boundaries, and treatment effect."

Reply:
@LittleBeannie thanks! Please see https://github.com/Merck/gsDesign2/pull/320/commits/f81a832e94a0f8bcfae3962761a38bc8e22b4c44